n_t/plots.py

```diff
@@ -1,12 +1,20 @@
 """
 Code for generating plots.
 """
+<<<<<<< HEAD
```

Comment:
Looks like we have some unresolved merge conflicts.

Reply:
wait-- ugh did something bad just happen here?

Reply:
will revert this merge and inspect

Reply:
actually this looks very minor. will fix right here.